code/game/machinery/doors/airlock.dm

```diff
@@ -1153,22 +1153,20 @@ About the new airlock wires panel:
 			if (shock(user, 75, I.siemens_coefficient))
 				user.delayNextAttack(10)
 
-	if((I.sharpness_flags & (CUT_AIRLOCK)) && user.a_intent == I_HURT && density)
+	if((I.sharpness_flags & (CUT_AIRLOCK)) && density)
 		user.visible_message("<span class='warning'>[user] begins slicing through \the [src]!</span>", \
 		"<span class='notice'>You begin slicing through \the [src].</span>", \
 		"<span class='warning'>You hear slicing noises.</span>")
 		playsound(src, 'sound/items/Welder2.ogg', 100, 1)
 
-		if(do_after(user, src, 200))
+		if(do_after(user, src, 100))
 			if(!istype(src))
 				return
 			user.visible_message("<span class='warning'>[user] slices through \the [src]!</span>", \
 			"<span class='notice'>You slice through \the [src].</span>", \
 			"<span class='warning'>You hear slicing noises.</span>")
 			playsound(src, 'sound/items/Welder2.ogg', 100, 1)
-			locked = 0
-			open()
-			operating = -1
+			bashed_in(user)
```

Comment:
This would somehow break the circuitry and unanchor the door. Are you sure you want to do this?

Reply:
To be fair battering ram can already do that

Reply:
That's the function these two share, yes.

But, why is a beam of pure light dealing such damage?

Reply:
I think it's because it's a star wars reference and it can cut pretty easily through a ton of things in the sw-universe